test/browser-specific/fixtures/esm.fixture.mjs

```diff
@@ -1,2 +1,3 @@
+/* eslint-disable-next-line import/no-absolute-path */
 import '/base/mocha.js';
```

Comment:
@boneskull I wonder the purpose of this line. When I tested, I have deleted the line, changed to refer to another file or missing file. But the test always passed. So I just left the linting issue by disabling it. Thank you for reading.

Reply:
`/base/mocha.js` is the path to the Mocha source as loaded by `karma-mocha` and served by karma's webserver.  we need to assert that we can import mocha via ESM in a browser, and this is the path where it's located when running under karma.

the eslint directive sounds appropriate, then.

Reply:
@boneskull Thanks a lot!